test/e2e/storage/persistent_volumes-local.go

```diff
@@ -325,30 +325,47 @@ var _ = SIGDescribe("PersistentVolumes-local [Feature:LocalPersistentVolumes] [S
 
 	Context("when using local volume provisioner", func() {
 		var volumePath string
+		var volumeContentDir string
 
 		BeforeEach(func() {
 			setupStorageClass(config, &immediateMode)
 			setupLocalVolumeProvisioner(config)
 			volumePath = path.Join(
 				hostBase, discoveryDir, fmt.Sprintf("vol-%v", string(uuid.NewUUID())))
+			volumeContentDir = path.Join(
+				hostBase, fmt.Sprintf("dir-%v", string(uuid.NewUUID())))
 		})
 
 		AfterEach(func() {
-			cleanupLocalVolumeProvisioner(config, volumePath)
+			cleanupLocalVolumeProvisioner(config, volumePath, volumeContentDir)
 			cleanupStorageClass(config)
 		})
 
 		It("should create and recreate local persistent volume", func() {
+
+			By("Creating a directory for content")
+			framework.Logf("creating local directory at path %q", volumeContentDir)
+			mkdirCmd1 := fmt.Sprintf("mkdir %v -m 777", volumeContentDir)
+			err := framework.IssueSSHCommand(mkdirCmd1, framework.TestContext.Provider, config.node0)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Creating a directory under discovery path for mounting")
+			framework.Logf("creating local directory at path %q", volumePath)
+			mkdirCmd2 := fmt.Sprintf("mkdir %v -m 777", volumePath)
+			err = framework.IssueSSHCommand(mkdirCmd2, framework.TestContext.Provider, config.node0)
+			Expect(err).NotTo(HaveOccurred())
+
+			By("Mounting directory under discovery path")
+			framework.Logf("mounting local directory %q at path %q", volumeContentDir, volumePath)
+			mntCmd := fmt.Sprintf("sudo mount --bind %v %v", volumeContentDir, volumePath)
```

Comment:
Would it be simpler to just have the content dir == mount dir?

Reply:
I'll try it out.

Reply:
@msau42 - PR updated.